msibi/optimize.py

```diff
@@ -136,6 +141,17 @@ def optimize(
             Number of iterations.
         start_iteration : int, default 0
             Start optimization at start_iteration, useful for restarting.
+        n_steps : int, default=1e6
+            How many steps to run the query simulations
+        integrator : str, default = "hoomd.md.integrate.nvt"
+            The integrator to use in the query simulation.
+            See hoomd-blue.readthedocs.io/en/v2.9.6/module-md-integrate.html
+        integrator_kwargs : dict, default = {"tau": 0.1}
+            The args and their values required by the integrator chosen
+        dt : float, default = 0.001
+            The time step delta
+        gsd_period : int, default = 1000
+            The frequency to write trajectory information to query.gsd
```

Comment:
these no longer have defaults, right?

Reply:
Good catch, just removed these from the doc strings.